MSAL/test/app/MSALTestAppSettings.m

```diff
@@ -99,7 +100,24 @@ + (void)initialize
     
     s_b2cAuthorities = @[signinPolicyAuthority, signupPolicyAuthority, profilePolicyAuthority];
     s_authorityTypes = @[@"AAD",@"B2C"];
-    s_profiles = _additionalProfiles();
+    
+    
+    NSString *defaultKey = @"MSAL-TestApp";
+    NSDictionary *defaultValue = @{@"clientId" : @"b6c69a37-df96-4db0-9088-2ab96e1d8215",
+                                   @"redirectUri" :@"msauth.com.microsoft.MSALTestApp://auth"};
+    
+    s_profiles = [[NSMutableDictionary alloc] initWithObjectsAndKeys:defaultValue, defaultKey, nil];
```

Comment:
nit: please use @{defaultKey:defaultValue} syntax

Reply:
This one is a mutable dictionary so keeping it this way.